Lib/mutatorMath/objects/mutator.py

```diff
@@ -133,7 +133,7 @@ def getAxisNames(self):
         s = {}
         for l, x in self.items():
             s.update(dict.fromkeys([k for k, v in l], None))
-        return s.keys()
+        return sorted(list(s.keys()))
```

Comment:
For robustness, otherwise I’d rather return a set which at least is explicitly unsorted. As I wrote earlier: 

> if you return a list and consumers expect a specific sorting, then we should give them better guarantees about this sorting rather than relying on the internals of the Python implementation. I wonder what PyPy’s behavior might be, for instance.